cloud-haven-react-app/src/components/Home/Home.js

```diff
@@ -1,13 +1,15 @@
 import React from 'react';
-import { TabSystem } from '../components';
+import { TabSystem, NotificationBar } from '../components';
+import { Container } from 'react-bootstrap';
 import './Home.css';
 
 function Home() {
 
    return (
-      <div>
+      <Container className="Home">
          <TabSystem/>
-      </div>
+         <NotificationBar/>
+      </Container>
```

Comment:
Right now on this branch, the notification bar isn't just on the home page. It's on each tab, though it's only on the home tab in the Figma. I believe this is because the TabSystem shouldn't actually be in this file. I can make a proposed plan for addressing this as a part of my task (the task for planning out the code architecture for the front end).